pulsar-client-cpp/lib/PatternMultiTopicsConsumerImpl.cc

```diff
@@ -33,7 +33,9 @@ PatternMultiTopicsConsumerImpl::PatternMultiTopicsConsumerImpl(ClientImplPtr cli
       patternString_(pattern),
       pattern_(std::regex(pattern)),
       autoDiscoveryTimer_(),
-      autoDiscoveryRunning_(false) {}
+      autoDiscoveryRunning_(false) {
+    namespaceName_ = TopicName::get(pattern)->getNamespaceName();
+}
```

Comment:
Wrong format?

Reply:
Did you mean code format?

Reply:
Yes. The indent looks weird.

Reply:
The C++ code format check automatically by CI, which uses `clang-format` and the `pulsar-client-cpp/.clang-format` to check the code style.

Reply:
okay